index.js

```diff
@@ -1,3 +1,43 @@
+const emojiObject = {
+  0: '\u0030\u20E3', 1: '\u0031\u20E3',
+  2: '\u0032\u20E3', 3: '\u0033\u20E3',
+  4: '\u0034\u20E3', 5: '\u0035\u20E3',
+  6: '\u0036\u20E3', 7: '\u0037\u20E3',
+  8: '\u0038\u20E3', 9: '\u0039\u20E3',
+  10: '\ud83d\udd1f', a: '\ud83c\udde6',
+  b: '\ud83c\udde7', c: '\ud83c\udde8',
+  d: '\ud83c\udde9', e: '\ud83c\uddea',
+  f: '\ud83c\uddeb', g: '\ud83c\uddec',
+  h: '\ud83c\udded', i: '\ud83c\uddee',
+  j: '\ud83c\uddef', k: '\ud83c\uddf0',
+  l: '\ud83c\uddf1', m: '\ud83c\uddf2',
+  n: '\ud83c\uddf3', o: '\ud83c\uddf4',
+  p: '\ud83c\uddf5', q: '\ud83c\uddf6',
+  r: '\ud83c\uddf7', s: '\ud83c\uddf8',
+  t: '\ud83c\uddf9', u: '\ud83c\uddfa',
+  v: '\ud83c\uddfb', w: '\ud83c\uddfc',
+  x: '\ud83c\uddfd', y: '\ud83c\uddfe',
+  z: '\ud83c\uddff', ' ': ' ',
+  '!': '\u2757', '?': '\u2753',
+  '#': '\u0023\ufe0f\u20e3','*': '\u002a\ufe0f\u20e3'
+};
+
+/**
+ * @typedef {Object} OptionsAndFlagsObject
+ * @property {Object} OptionsAndFlagsObject.options All the parsed options.
+ * @property {string[]} OptionsAndFlagsObject.flags All the parsed flags.
+ * @property {string} OptionsAndFlagsObject.contentNoOptions All the provided strings in the array concatenated without the options.
+ * @property {string} OptionsAndFlagsObject.contentNoFlags All the provided strings in the array concatenated without the flags.
+ */
```

Comment:
```suggestion
/**
 * @typedef {Object} OptionsAndFlagsObject
 * @property {Object} options All the parsed options.
 * @property {string[]} flags All the parsed flags.
 * @property {string} contentNoOptions All the provided strings in the array concatenated without the options.
 * @property {string} contentNoFlags All the provided strings in the array concatenated without the flags.
 */
```
nit: these can be shortened to this c:

Reply:
oki, can do! <3